ndb/tests/unit/test_model.py

```diff
@@ -584,6 +585,221 @@ def test___pos__():
         with pytest.raises(NotImplementedError):
             +prop
 
+    @staticmethod
+    def test__do_validate(property_clean_cache):
```

Comment:
How does that fixture get to be available in this module?  Is `conftest.py` a majyk thing in pytest-land?

Reply:
Yes `conftest.py` is a standard part of `pytest`. FWIW `pytest` is widely used in the Python community.

Reply:
I know it is widely used -- that doesn't mean it doesn't have warts.  Majyk names injected into testcase modules is pretty icky.